FunctionalProgramming/Recursion/SuperDigit.fsx

```diff
@@ -1,37 +1,68 @@
 ﻿open System
 
-let splitBySpace (input:string) =
-    input.Split ' '
+// I generally put such tiny functions on single lines; this saves space and
+// improves the reading flow, because a piece of code such as this function
+// body can get a bit "lost" visually.
+let splitBySpace (input : string) = input.Split ' '
 
-let readInputs _ =
-    let inputs = 
-        Console.ReadLine () 
-        |> splitBySpace 
-    inputs.[0], int64 inputs.[1]
+// While this works with `_`, the placeholder parameter of parameterless
+// functions is by convention always the `unit` value `()`. That makes it
+// immediately clear that there is no useful input to this function, while the
+// `_`  placeholder actually causes the parameter to become generic, which is
+// irritating, because while one can pass *anything*, it will never matter.
+let readInputs () =
```

Comment:
That's good to know.  I originally had it without any parameter and it didn't work.  I found out that if I put the _ in as a parameter then it worked.  I didn't know about the ().  I will use that from now on.